project/src/components/review-form/review-form.tsx

```diff
@@ -0,0 +1,84 @@
+import { FormEvent, useState } from 'react';
+
+function ReviewForm(): JSX.Element {
+  const [review, setReview] = useState('');
+  const [completedReview, setCompletedReview] = useState(false);
+
+  const handleSubmit = (e: FormEvent) => {
+    e.preventDefault();
+    // eslint-disable-next-line no-alert
+    alert(`Your review: ${review}`);
+    setReview('');
+    setCompletedReview(false);
+  };
+  const handleReviewChange = (e: FormEvent<HTMLTextAreaElement>) => {
+    setReview(e.currentTarget.value);
+
+    review.length >= 50 ? setCompletedReview(true) : setCompletedReview(false);
+  };
+
+  return (
+    <form className="reviews__form form" onSubmit={handleSubmit}>
+      <label className="reviews__label form__label" htmlFor="review">Your review</label>
+      <div className="reviews__rating-form form__rating">
+        <input className="form__rating-input visually-hidden" name="rating" value="5" id="5-stars" type="radio" />
+        <label htmlFor="5-stars" className="reviews__rating-label form__rating-label" title="perfect">
+          <svg className="form__star-image" width="37" height="33">
+            <use xlinkHref="#icon-star" />
+          </svg>
+        </label>
+
+        <input className="form__rating-input visually-hidden" name="rating" value="4" id="4-stars" type="radio" />
+        <label htmlFor="4-stars" className="reviews__rating-label form__rating-label" title="good">
+          <svg className="form__star-image" width="37" height="33">
+            <use xlinkHref="#icon-star" />
+          </svg>
+        </label>
+
+        <input className="form__rating-input visually-hidden" name="rating" value="3" id="3-stars" type="radio" />
+        <label htmlFor="3-stars" className="reviews__rating-label form__rating-label" title="not bad">
+          <svg className="form__star-image" width="37" height="33">
+            <use xlinkHref="#icon-star" />
+          </svg>
+        </label>
+
+        <input className="form__rating-input visually-hidden" name="rating" value="2" id="2-stars" type="radio" />
+        <label htmlFor="2-stars" className="reviews__rating-label form__rating-label" title="badly">
+          <svg className="form__star-image" width="37" height="33">
+            <use xlinkHref="#icon-star" />
+          </svg>
+        </label>
+
+        <input className="form__rating-input visually-hidden" name="rating" value="1" id="1-star" type="radio" />
+        <label htmlFor="1-star" className="reviews__rating-label form__rating-label" title="terribly">
+          <svg className="form__star-image" width="37" height="33">
+            <use xlinkHref="#icon-star" />
+          </svg>
+        </label>
+      </div>
+      <textarea
+        className="reviews__textarea form__textarea"
+        id="review"
+        name="review"
+        placeholder="Tell how was your stay, what you like and what can be improved"
+        value={review}
+        onChange={(e) => handleReviewChange(e)}
```

Comment:
```suggestion
        onChange={handleReviewChange}
```

Он по дефолту будет получать - e